taichi/transforms/ir_printer.cpp

```diff
@@ -825,6 +825,16 @@ class IRPrinter : public IRVisitor {
     print(result);
   }
 
+  void visit(GetElementStmt *stmt) override {
+    std::string index = "";
+    for (int i = 0; i < stmt->index.size(); i++) {
```

Comment:
We can use `fmt::join(stmt->index, ", ")` here, much cleaner

@k-ye Is it correct that we will very likely keep `spdlog` and `libfmt` a part of the compiler infra?